joint_limits:
  left_wheel_joint:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false
  right_wheel_joint:
    has_position_limits: false
    has_velocity_limits: false
    max_velocity: 1.0
    has_acceleration_limits: false
    has_jerk_limits: false
    has_effort_limits: false